src/backends/interflop-mca/interflop_mca.c

```diff
@@ -196,11 +213,11 @@ static __float128 _noise_binary128(const int exp) {
 /* Macro function for checking if the value X must be noised */
 #define _MUST_NOT_BE_NOISED(X, VIRTUAL_PRECISION)                              \
   /* if mode ieee, do not introduce noise */                                   \
-  (MCALIB_MODE == mcamode_ieee) ||					\
-  /* Check that we are not in a special case */				\
-  (FPCLASSIFY(X) != FP_NORMAL && FPCLASSIFY(X) != FP_SUBNORMAL) ||	\
-  /* In RR if the number is representable in current virtual precision, */ \
-  /* do not add any noise if */						\
+  (MCALIB_MODE == mcamode_ieee) ||					                                   \
+  /* Check that we are not in a special case */				                         \
+  (FPCLASSIFY(X) != FP_NORMAL && FPCLASSIFY(X) != FP_SUBNORMAL) ||	           \
+  /* In RR if the number is representable in current virtual precision, */     \
+  /* do not add any noise if */						                                     \
```

Comment:
Please avoid white-space only changes.

Reply:
Ok.

Reply:
This must have happened when clang-format complained...